### PR TITLE
Assignment9+10 - Nolan Posey

### PR DESCRIPTION
**Pull request by Nolan-Posey**

### CODE REVIEW

**Review by BenFristadSchool — COMMENTED** [review]
I will review this

File: Assignment/Directory.Build.props [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <Project>
 <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
```
Comment (BenFristadSchool) [inline Assignment/Directory.Build.props:3]: should be 7.0

File: Assignment/Assignment/PingProcess.cs [hunk 2]
```diff
@@ -27,40 +28,51 @@ public PingResult Run(string hostNameOrAddress)
...
     {
-        throw new NotImplementedException();
+        Task<PingResult> task = Task.Factory.StartNew(() => Run(hostNameOrAddress));
+        
+        return task;
     }
 
     async public Task<PingResult> RunAsync(
-        string hostNameOrAddress, CancellationToken cancellationToken = default)
+        string hostNameOrAddress, params CancellationToken[] cancellationToken)
```
Comment (BenFristadSchool) [inline Assignment/Assignment/PingProcess.cs:37]: cancelationToken can be simplified to be an object rather than an object array

**Review by BenFristadSchool — COMMENTED** [review]
Well Done!

**Comment by mmwoodfo** [comment]
Looks like you had some good PR comments

**Pull request closed without merging**